lib/modules/manager/bundler/artifacts.ts

```diff
@@ -149,47 +150,12 @@ export async function updateArtifacts(
       {} as Record<string, string>,
     );
 
-    // Detect hosts with a hyphen '-' in the url.
-    // Those cannot be added with environment variables but need to be added
-    // with the bundler config
-    const bundlerHostRulesAuthCommands: string[] = bundlerHostRules.reduce(
-      (authCommands: string[], hostRule) => {
-        if (hostRule.resolvedHost?.includes('-')) {
-          // TODO: fix me, hostrules can missing all auth
-          const creds = getAuthenticationHeaderValue(hostRule);
-          authCommands.push(`${quote(hostRule.resolvedHost)} ${quote(creds)}`);
-        }
-        return authCommands;
-      },
-      [],
-    );
-
     const bundler = getBundlerConstraint(
       updateArtifact,
       existingLockFileContent,
     );
     const preCommands = ['ruby --version'];
 
-    // Bundler < 2 has a different config option syntax than >= 2
```

Comment:
Is this PR dropping support for Bundler v1 too? I think it should be a separate PR if so.

Reply:
As far as I can tell, the bundler v1 distinction is only needed here because the bundler authentication command in v1 differs from v2, but when using env vars the command is not needed anymore and works the same for both.